src/store/traction/pacbio/libraries/actions.js

```diff
@@ -82,63 +83,6 @@ const setLibraries = async ({ commit, getters }, filter, page) => {
   return { success, errors, meta }
 }
 
-const updateTag = async ({ getters }, payload) => {
```

Comment:
I presume these methods are no longer used?

Reply:
Yeah they aren't used. I think they were from when you could edit a library individually and not through a pool, but it would be best to remove them to prevent confusion.